src/main/java/school/lemon/changerequest/java/vehicles/Vehicle.java

```diff
@@ -0,0 +1,16 @@
+package school.lemon.changerequest.java.vehicles;
+
+/**
+ * Created by Yaroslav Pavlinskiy on 05.12.2016.
+ */
+public abstract class Vehicle {
```

Comment:
this class could implement `IVehicle` and you should move all getter/setters here, so that you won't duplicate code and could also make all variables private.